Global:
  g: -9.801

Estimator:
  W_sigma:
    sigma_omega_m: [5.0e-2, 5.0e-2, 5.0e-2]
    sigma_b_g: [1.0e-5, 1.0e-5, 1.0e-5]
    sigma_a_m: [1.0e-1, 1.0e-1, 1.0e-1]
    sigma_b_a: [1.0e-4, 1.0e-4, 1.0e-4]
  N_sigma:
    sigma_mocap_p: [1.0e-3, 1.0e-3, 1.0e-3]
    sigma_mocap_q: [1.0e-4, 1.0e-4, 1.0e-4, 1.0e-4]
